extensions/azurecore/package.json

```diff
@@ -33,12 +33,43 @@
       },
       {
         "type": "object",
-        "title": "Azure Account Configuration",
+        "title": "%config.azureAccountConfigurationSection%",
         "properties": {
-          "accounts.azure.enablePublicCloud": {
+          "accounts.azure.cloud.enablePublicCloud": {
             "type": "boolean",
             "default": true,
             "description": "%config.enablePublicCloudDescription%"
+          },
+          "accounts.azure.cloud.enableUsGovCloud": {
+            "type": "boolean",
+            "default": false,
+            "description": "%config.enableUsGovCloudDescription%"
+          },
+          "accounts.azure.cloud.enableGermanyCloud": {
```

Comment:
I'd make it clear: GermanyGovCloud and ChinaGovCloud so that users won't think it is the public data centers there. similarly the descriptions should be updated accordingly.

Reply:
Is it just government? Can businesses in China use public Azure?

Reply:
I'll resolve the comment, they are not Government cloud.